frontend/src/hooks/useExportCsvEffect.ts

```diff
@@ -2,26 +2,33 @@ import { useState } from 'react'
 import { CSVReportRequestDTO } from '@src/clients/report/dto/request'
 import { csvClient } from '@src/clients/report/CSVClient'
 import { ERROR_MESSAGE_TIME_DURATION } from '@src/constants'
+import { AxiosError } from 'axios'
```

Comment:
AxiosError is from dep's lib. should warp it in httpClient with some exceptions. Do not use it directly, keep `Layered Architecture`